src/gamestate/commands.cpp

```diff
@@ -3844,6 +3844,62 @@ void execute_toggle_rebel_hunting(sys::state& state, dcon::nation_id source, dco
 	}
 }
 
+void toggle_select_province(sys::state& state, dcon::nation_id source, dcon::province_id prov) {
+	payload p;
+	memset(&p, 0, sizeof(payload));
+	p.type = command_type::toggle_select_province;
+	p.source = source;
+	p.data.generic_location.prov = prov;
+	add_to_command_queue(state, p);
+}
+bool can_toggle_select_province(sys::state& state, dcon::nation_id source, dcon::province_id prov) {
+	if(state.world.province_get_nation_from_province_control(prov) != source)
+		return false;
```

Comment:
while the control condition does match the condition for buildings, my guess is that it is unwanted by the mod makers. They probably want to be able to select occupied provinces